### DISCUSSION TITLE
Only Suricata, Zeek  and strelka logs forwarding to elastic stack 7.17.1

### DISCUSSION THREAD

**Discussion by Blason** [Security-Onion-Solutions/securityonion#8076] [Unsupported Versions] [answered]
Hi Team,

I wanted to install only security onion forward nodes on one of my ubuntu hardware or using ISO image. I already have Elastic SIEM running with version 7.17.1 and I am keen integrating zeek/suricata and strelka to elasticsearch. can someone please suggest the way?

Since on my elastic stack I already have filebeat running, can I forward the logs to filebeat with syslog module? Or any other method?

TIA
Blason R

**Comment by dougburks** [comment, marked as answer]
Most organizations use Security Onion to build their entire deployment (including the Elastic Stack) so that they can take advantage of all of our pre-built configuration and parsing.

If you want to send logs outside of Security Onion, please see:
https://docs.securityonion.net/en/2.3/syslog-output.html
